### PR TITLE
added container networking diagnostics page

### PR DESCRIPTION
**Pull request by daschott**
@JMesser81 @dineshgovindasamy please LGTM.

### CODE REVIEW

File: Diagnostics/HNS_diagnostics.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+# Container Networking Diagnostics
+
+> If you face DNS or networking issues with containers, **the best solution** is to ensure you are running the _**most recent Windows release**_. We worked  very hard to iron out container networking issues in Windows 1803, so we encourage you to update and benefit from the smoother experience provided by the newest releases.
```
Comment (JMesser81) [inline Diagnostics/HNS_diagnostics.md]: Similar to changes I made here: https://hackmd.io/hpKY9Pn_TSKRO_3WWyEjdQ?both#  please ensure we call out this is for Windows Server, version XXXX and Windows 10 Creators Update

**Comment by JMesser81** [comment]
Made some comments

File: Diagnostics/HNS_diagnostics.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (daschott) [reply Diagnostics/HNS_diagnostics.md]: https://github.com/Microsoft/SDN/blob/a8a9173f1d0eddf530535bfdbdfeb19d57056428/Diagnostics/ContainerDiagnostics.md should be the right one

**Pull request merged**